.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "8"
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+install:
+  - npm version
+  - npm install
+script:
+  - npm run build
```

Comment:
This is to test building process right?

Reply:
yes. This is to build before we run tests